roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -15,7 +15,7 @@ auth service required = none
 auth client required = none
 auth supported = none
 {% endif %}
-fsid = {{ fsid }}
+{% if not mon_containerized_deployment_with_kv%}fsid = {{ fsid }}{% endif %}
```

Comment:
Now i just mount ceph.conf as /etc/ceph/ceph.defaults) into populate docker and let him  push this into etcd. Add a flag for a user to decide if use default ceph-docker/ceph.conf or to push custom one ?